2_horseshoe_bat/api_definitions/rest/metadata_repository.yaml

```diff
@@ -0,0 +1,3886 @@
+          title: Has Member
+        main_contact:
+          description: The main contact for the Data Access Committee.
+          title: Main Contact
+          type: string
+        name:
+          description: The name for an entity.
+          title: Name
+          type: string
+        xref:
+          description: Alternate identifiers for an entity.
+          items:
+            type: string
+          title: Xref
+          type: array
+      title: CreateDataAccessCommittee
+      type: object
+    CreateDataAccessPolicy:
+      properties:
+        description:
+          description: A short description for the Data Access Policy.
+          title: Description
+          type: string
+        ega_accession:
+          description: A unique European Genome-Phenome Archive (EGA) identifier             assigned
+            to an entity for the sole purpose of referring to that entity             within
+            the EGA federated network.
+          title: Ega Accession
+          type: string
+        has_data_access_committee:
+          anyOf:
+          - type: string
+          - $ref: '#/components/schemas/CreateDataAccessCommittee'
+          description: The Data Access Committee linked to this policy.
+          title: Has Data Access Committee
+        has_data_use_condition:
+          anyOf:
+          - type: string
+          - items:
+              $ref: '#/components/schemas/DataUseCondition'
+            type: array
+          description: Data Use Condition entities that are associated with the             Data
+            Access Policy.
+          title: Has Data Use Condition
+        name:
+          description: The name for an entity.
+          title: Name
+          type: string
+        policy_text:
+          description: The terms of data use and policy verbiage should be             captured
+            here.
+          title: Policy Text
+          type: string
+        policy_url:
+          description: URL for the policy, if available. This is useful if the             terms
+            of the policy is made available online at a resolvable URL.
+          title: Policy Url
+          type: string
+        xref:
+          description: Alternate identifiers for an entity.
+          items:
+            type: string
+          title: Xref
+          type: array
+      title: CreateDataAccessPolicy
+      type: object
+    CreateDataset:
+      properties:
+        has_data_access_policy:
+          anyOf:
+          - type: string
+          - $ref: '#/components/schemas/CreateDataAccessPolicy'
+          description: The Data Access Policy that applies to this Dataset.
+          title: Has Data Access Policy
+        has_file:
+          anyOf:
+          - items:
+              type: string
+            type: array
+          - items:
+              $ref: '#/components/schemas/CreateFile'
+            type: array
```

Comment:
Files should only be created using submission objects, right?
Should we really allow to post new Files here?
Probably it's good to just allow an ID to an already existing file here. Don't you agree?